rightteaminc/rightstyle-js
# PV-66 Side drawer component

Ref: https://pollywog.atlassian.net/browse/PV-66

Note:every PR needs a ticket

### Issue
``We needed a custom side drawer component ``


### Solution
``1.) Created a demo Drawer page. 2.) Added a custom drawer component to rightstyle-js and consumed it in the demo page.``


### Testing Notes: n/a

### QA Notes
QA Resource:

Steps to reproduce and/or verify changes
  * Step 1: Switch to the Side-Menu branch
  * Step 2: Run rightstyle-js locally

### Side Effects
- [ ] Does this change introduce breaking changes to apis, etc? *If so or not sure, please communicate to team for [semver](https://semver.org/) bump*
- [ ] Did working on this ticket spawn other tickets, tech debt, or cruft. If so, please list ticket numbers.

### Dev Checklist
 - [x] There is a Jira Ticket for this changes
 - [x] The Jira ticket is assigned to me
 - [x] The Jira ticket is pointed
 - [x] The Jira Ticket is on the [board](https://pollywog.atlassian.net/secure/RapidBoard.jspa?projectKey=PV) and "In Code Review"
 - [ ] I have run automated tests and manually tested my change prior to tagging others for code review
 - [ ] I have run `make build` on the last commit (if applicable)
 - [ ] All TODO comments have associated tickets

### Code Reviewer Checklist
- I have reviewed the code changes +1
- I have checked out the code and run it locally +10
- I have deployed the code and manually tested +100
- I have verified the Jira ticket associated with this PR is correct and has required fields

## Merging Checklist
- Do not not squash without checking
- Please delete the branch on merged
- Ensure Jira ticket is closed and in "Done" column on the [board](https://pollywog.atlassian.net/secure/RapidBoard.jspa?projectKey=PV)

Note: Pollywog PR template v0.1.0 - If making changes, please consider updating the other repositories as well
